pulumi/integration_tests/__main__.py

```diff
@@ -87,7 +87,7 @@ def main() -> None:
         "aws_env_vars_for_local": grapl_stack.aws_env_vars_for_local,
         "aws_region": aws.get_region().name,
         "container_images": _integration_new_container_images(artifacts),
-        "rust_log": log_levels.RUST_LOG_LEVELS,
+        "dns_server": config.CONSUL_DNS_IP,
```

Comment:
another Weird Thing That's Here. I think we definitely want that rust_log

Reply:
fixed